integration-tests/constants.go

```diff
@@ -0,0 +1,35 @@
+package integrationtests
+
+import (
+	"math/big"
+
+	"github.com/0xProject/0x-mesh/constants"
+)
+
+const (
+	ethereumRPCURL  = "http://localhost:8545"
+	ethereumChainID = 1337
+	rpcPort         = 60501
+
+	// Various config options/information for the bootstrap node. The private key
+	// for the bootstrap node is checked in to version control so we know it's
+	// peer ID ahead of time.
+	bootstrapAddr    = "/ip4/127.0.0.1/tcp/60500/ws"
+	bootstrapPeerID  = "16Uiu2HAmGd949LwaV4KNvK2WDSiMVy7xEmW983VH75CMmefmMpP7"
+	bootstrapList    = "/ip4/127.0.0.1/tcp/60500/ws/ipfs/16Uiu2HAmGd949LwaV4KNvK2WDSiMVy7xEmW983VH75CMmefmMpP7"
+	bootstrapDataDir = "./data/bootstrap-0"
+
+	// Various config options/information for the standalone node. Like the
+	// bootstrap node, we know the private key/peer ID ahead of time.
```

Comment:
This comment doesn't match the underlying code and also doesn't hold true anymore. We don't always know the peer ID ahead of time for all the standalone nodes that are started in the integration tests.